OneCommunityGlobal/HighestGoodNetworkApp
# Aishwarya review only --> Alan mark task done dashboard

This is a PR that addresses the problem when we mark a task as done the remaining hours are not deducted.
How to test:
- Log in as admin and mark a task as done.
- Check their original remaining hours.
- Check their remaining hours after marking the task as done.

Please watch this video.

https://user-images.githubusercontent.com/67836976/236922931-0934792d-4948-42e5-87c6-89673be1e3a5.mov
